Documentation/gettingstarted/host-firewall.rst

```diff
@@ -0,0 +1,156 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _host_firewall:
+
+********************
+Host Firewall (beta)
+********************
+
+This document serves as an introduction to Cilium's host firewall, to enforce
+security policies for Kubernetes nodes.
+
+.. include:: ../beta.rst
+
+Enable the Host Firewall in Cilium
+==================================
+
+.. include:: k8s-install-download-release.rst
+
+Deploy Cilium release via Helm:
+
+.. parsed-literal ::
+
+    helm install cilium |CHART_RELEASE|        \\
+      --namespace kube-system                  \\
+      --set global.kubeProxyReplacement=strict \\
+      --set global.hostFirewall=true           \\
+      --set global.devices=ethX,ethY
+
+At this point, the Cilium-managed nodes are ready to enforce network policies.
+
+
+Enable Policy Audit Mode for the Host Endpoint
+==============================================
+
+When applying `HostPolicies` carelessly, it's easy to block the access to the
+nodes or to break the cluster's normal behavior (ex. by blocking communication
+with kube-api).
+
+To avoid such issues, we can switch the host firewall in audit mode, to
+validate the impact of host policies before enforcing them.
```

Comment:
Probably worth making it clear that policy audit mode disables policy enforcement, with similar wording to the policy GSG.

```suggestion
To avoid such issues, we can switch the host firewall in audit mode, to
validate the impact of host policies before enforcing them. When Policy Audit Mode is enabled, no network policy is enforced so this setting is *not recommended for production deployment*.
```

(I appreciate that we're probably moving folks from no security -> hostfw so it's not such a big deal. I also note that the instructions here use the `endpoint config` approach to individually enable policy audit for the host which mitigates the risk to policy enforcement for pods. So if you want to be less loud with the warning I could understand, above is suggestion taken from the other doc).

Reply:
I think that's light enough, I'll add it.

And yes, I took some time to write this because I wanted to be able to switch into policy audit mode only for the host firewall. Otherwise, we can't expect users to take this approach in more realistic environments. The downside is that it exposes `cilium endpoint config` a bit more than we've used to in the past.